showcase/src/app/shared/example-viewer/example-viewer.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { Observable } from "rxjs";
+
+@Injectable()
+export class ExampleViewerService {
+	public constructor(private http: HttpClient) {}
+
+	public fetchFileAlt(path: string): Observable<string> {
```

Comment:
Question... Why does the name of this method ends with "Alt" ? Perhaps is simpler to call it fetchFile() only.